Dockerfile

```diff
@@ -13,7 +13,7 @@
 FROM golang:alpine AS build-env
 
 # Install minimum necessary dependencies,
-ENV PACKAGES curl make git libc-dev bash gcc linux-headers eudev-dev python3
+ENV PACKAGES curl make git libc-dev bash gcc linux-headers eudev-dev python3 plantuml
```

Comment:
@hjorthjort could you add a way to generate the plantuml? This is incorrect

Reply:
@marbar3778 Can you elaborate on "a way to generate the plantuml"? We need a plantuml binary to generate diagrams. By far the least hairy way would be to install it as a package from the standard package manager. Is it not available?

The other options are building it from source (hairier) or committing the official `.jar` directly, which would not stay up to date. The `.jar` option might be best if we can't install the package.

Reply:
Another option would be to do something like this: https://github.com/miy4/docker-plantuml/blob/master/Dockerfile

Reply:
It seems you were trying to generate the uml files for useage in docs. This was failing because there is no uml binary. Is it okay we just dont generated the uml files into something that can be rendered?

Reply:
It's also possible to just generate the files locally (by anyone who has plantuml) and we simply commit the files. Not having them in the docs at all seems like a big step back, they are there to clarify things and there seemed to be a lot of positive feedback on having them.

But it's up to you what you think is the most hygenic approach.

Reply:
we can generate them locally and push them. Could you do this? I dont see us changing these files

Reply:
okay. can you update your open PR to remove the plantuml from the Dockerfile and remove (instead of commenting out) the line for generating them.

Reply:
This should be solved by https://github.com/cosmos/cosmos-sdk/pull/9124#issuecomment-820646522